### PR TITLE
Updated code that reads data from zarr files

### PR DESCRIPTION
**Pull request by PranjalSahu**

### CODE REVIEW

**Review by thewtex — APPROVED** [review]

File: oai_analysis_2/dask_processing.py [hunk 1]
```diff
@@ -2,190 +2,192 @@
 
 import numpy as np
 import os
-#os.environ["CUDA_VISIBLE_DEVICES"]=""
 
+# os.environ["CUDA_VISIBLE_DEVICES"]=""
 
 import dask
 from dask import delayed, compute, visualize
 from dask.distributed import Client, progress, LocalCluster
```
Comment (thewtex) [inline oai_analysis_2/dask_processing.py:10]: remove

**Comment by HastingsGreer** [comment]
The ITK issue causing the failing test:
https://github.com/InsightSoftwareConsortium/ITK/issues/3592

**Comment by thewtex** [comment]
@HastingsGreer over the next few months, we will be working on upgrading the packaging, which will hopefully address manifest issues or we will be able to dive deeper at that time. In the meantime, can the environment running the packages please get a fresh install?

**Comment by HastingsGreer** [comment]
I wiped the test server and am re-running.

**Pull request merged**